### PR TITLE
[TASK] Create NZD funding page (RT-3060)

### PR DESCRIPTION
**Pull request by Mesrop**

### CODE REVIEW

File: src/jade/tabs/nzd.jade [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          .title(l10n) Deposit Options
+          .description(l10n) 
+            | Coinex accepts deposits by the following methods:
+            ul 
+              li
+                p(l10n) International bank transfer
+              li
+                p(l10n) Ripple transaction
+              li
+                p(l10n) Stellar transaction
```
Comment (mrajvanshy) [inline src/jade/tabs/nzd.jade]: Do we want to advertise Stellar?

Comment (annatonger) [inline src/jade/tabs/nzd.jade]: yes, it's fine :)

On Thu, Jan 15, 2015 at 12:14 PM, Malika Rajvanshy <notifications@github.com

**Comment by mrajvanshy** [comment]
LGTM

**Comment by yongsoo** [comment]
LGTM

**Pull request merged**